projects/igniteui-angular/migrations/common/tsUtils.ts

```diff
@@ -185,7 +185,7 @@ const patchHostOverwrite = (host: Tree, fileVersions: Map<string, number>) => {
  */
 export const createProjectService = (serverHost: tss.server.ServerHost): tss.server.ProjectService => {
     // set traceToConsole to true to enable logging
-    const logger = new Logger(false, tss.server.LogLevel.verbose);
+    const logger = new Logger(true, tss.server.LogLevel.verbose);
```

Comment:
Might want to set that back to `false` so we don't flood the console with too many messages.